hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -131,8 +132,23 @@ public void triggerContainerHealthCheck() {
       LOG.info("Container Health task thread took {} milliseconds to" +
               " process {} existing database records.",
           Time.monotonicNow() - start, existingCount);
+
+      checkAndProcessContainers(unhealthyContainerStateStatsMap, currentTime);
+      processedContainers.clear();
+    } finally {
+      lock.writeLock().unlock();
+    }
+  }
+
+  private void checkAndProcessContainers(
+      Map<UnHealthyContainerStates, Map<String, Long>>
+          unhealthyContainerStateStatsMap, long currentTime) {
+    ContainerID startID = ContainerID.valueOf(1);
+    List<ContainerInfo> containers = containerManager.getContainers(startID,
+        Integer.parseInt(DEFAULT_FETCH_COUNT));
```

Comment:
Could we replace the usage of `Integer.parseInt(DEFAULT_FETCH_COUNT)` with a named constant or a method as `DEFAULT_FETCH_COUNT` is a constant. 

```
private static final int FETCH_COUNT = Integer.parseInt(DEFAULT_FETCH_COUNT);
```

Reply:
Ok.